docs/wallet/how-to-join-arbitrum-and-see-your-tokens.mdx

```diff
@@ -0,0 +1,132 @@
+:::
+
+### Add Arbitrum Network to Metamask
+
+Adding Arbitrum Network support to MetaMask is a mandatory step in order to add the WXM token, view activity, and make transactions.
+Make sure you follow our step-by-step guide below for adding the Arbitrum One Network to MetaMask.
+
+We will provide guidance for both Arbitum Mainnet and Testnet (also called Sepolia Testnet) but
+as we are currently in testnet, **you need to add the Testnet network.**
+
+
+<hr/>
+
+### Using Arbiscan
+
+:::note
+Please make sure you have already installed **[MetaMask](https://metamask.io/)**!
+
+**This solution works only on desktop with the MetaMask browser extension already installed.**
+:::
+
+<Tabs
+  defaultValue="mainnet"
+  values={[
+    { label: 'Arbitrum One Mainnet', value: 'mainnet', },
+    { label: 'Arbitrum One Testnet (Sepolia)', value: 'Sepolia', },
+  ]
+}>
+
+<TabItem value="mainnet">
+
+:::note
+This is just for informational purposes. At this stage we are currently in testnet so **you will need to add the Testnet network.**
+:::
+
+Please follow the steps to add Arbitrum One Mainnet:
+
+- Navigate to [Arbiscan.io](https://arbiscan.io/)
+- Scroll down to the bottom of the page and click on the button `Add Arbitrum One Network`
+
+<img class="marginTopBottomNormal" src={useBaseUrl("/img/wallet/Arbitrum-One.png")} alt="Arbitrum One" />
+
+- Once you click the button you will see a MetaMask notification, now click on **Approve**.
+You will be directed to Arbitrum's Mainnet in the network dropdown list. You can now close the dialog.
+
+<img class="marginTopBottomNormal" src={useBaseUrl("/img/wallet/approve_on_metamask.png")} />
+
+
+<hr/>
+
+:::tip
+Alternatively you can add Arbitrum One Mainnet by following the steps below:
+
+Visit: [Chainlist](https://chainlist.org/chain/42161) and then click on the button `Add to Metamask`.
+:::
+
+</TabItem>
+
+<TabItem value="Sepolia">
+
+Please follow the steps to add Arbitum one Testnet (also known as Sepolia Testnet):
+
+1. Make sure that you have successfully added Arbitrum One Mainnet to your Metamask.
+2. Open Metamask and look for the network selection tool located at the upper left corner.
+3. Enable the "Show Test Networks" option.
+4. From the dropdown list, select "Sepolia" test network.
+
+<img class="marginTopBottomNormal" src={useBaseUrl("/img/wallet/add_sepolia_testnet.png")} />
+
+:::tip 
+Alternatively you can add Arbitrum One Testnet by following the steps below:
+
+Visit the official [Arbitrum One Testnet](https://testnet.arbiscan.io/) and then click on the button `Add to Metamask`.
+
+or
+
+Visit the official Documentation [Arbitrum One Testnet](https://developer.offchainlabs.com/docs/developer_quickstart) and then click on the button `Add to Metamask`.
+:::
+</TabItem>
+
+</Tabs>
+
+
+
+<hr/>
+
+
+
+### Add WXM token in MetaMask
+
+For the WXM token to appear in the wallet UI, you will need to add it manually.
```

Comment:
"WXM" token here, but symbol below is "MWX" ?

Reply:
fixed